### ISSUE TITLE
Full library Search From search option

### ISSUE THREAD

**Issue by Gladius-007** [inorichi/tachiyomi#3800]
---

#Why this feature?
(This feature will be very advantageous for the users to search all the comics in library even if you have filtered them by the given options.)

#What this feature will do?
Right now, in v0.10.5, suppose if we have filtered the comics by downloaded option then if I use the search option on upper right corner and search a comic, which is in the library but not filtered library, it won't show up. I request the feature that would enable the user to search the full library when search option is used.

**Comment by lenisko** [comment]
Yeah it would be great to get an option to ignore filters for library search.

**Comment by Sam-Oak** [comment]
maybe it would be nice if i could also search for manga in other categories

**Comment by arkon** [comment]
Merging with #4493 -- would probably make the most sense to show an "All" category first when searching that isn't affected by filters.
